tests/integration/jinad/test_simple_distributed/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.7.6-slim
+
+WORKDIR /
+
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y git \
+                                               curl
+
+COPY tests/integration/test_simple_distributed/requirements.txt /tests/integration/test_simple_distributed/requirements.txt
+
+RUN python -m pip install --no-cache-dir --upgrade pip && \
+    pip install --no-cache-dir -r tests/integration/test_simple_distributed/requirements.txt
+
+RUN git clone https://github.com/jina-ai/jinad.git && \
+    pip install --no-cache-dir -r jinad/requirements.txt
```

Comment:
I think we wanted to use the `jinad` image instead of installing `jinad` here. Or did this change?

Reply:
Here, we are installing `jinad` with requirements, but using the `test-pip` version of jina for requirement, for consistency.